lib/proto.js

```diff
@@ -143,23 +143,20 @@ function setBindings (newBindings) {
 function write (_obj, msg, num) {
   const t = this[timeSym]()
   const mixin = this[mixinSym]
-  const objError = _obj instanceof Error
   var obj
 
   if (_obj === undefined || _obj === null) {
     obj = mixin ? mixin() : {}
-  } else {
-    obj = Object.assign(mixin ? mixin() : {}, _obj)
-    if (!msg && objError) {
+  } else if (_obj instanceof Error) {
```

Comment:
what versions did you find there was no slow down? I thought instanceof was expensive? Is that no longer true in 12 and 14?

Reply:
`instanceof` was already there. Now there is even a branching being skipped in opposition to the original code.